tests/test-import.js

```diff
@@ -18,6 +18,24 @@ describe('layout: importer', function () {
   const dirBig = __dirname + '/test-data/dir-big'
   const dirNested = __dirname + '/test-data/dir-nested'
 
+  // check to see if missing empty dirs need to be created
+  fs.exists(__dirname+'/test-data/dir-nested/dir-another', function(exists){
+    if(!exists){
+      fs.mkdir(__dirname+'/test-data/dir-nested/dir-another', function(){
+        fs.exists(__dirname+'/test-data/dir-nested/level-1/level-2', function(exists){
```

Comment:
ahh ty for pointing me to fs.stat, didn't realize exists was the deprecated version, will change for a sync version, also was checking twice on the level-2 to cover a weird case that someone would have dir-another but not level-2 or vice versa. Either way that should only be executed when the repo is cloned so it could be assumed both won't exist.